src/main/java/com/shippingcostestimator/enterprise/dao/ShipmentDAOStub.java

```diff
@@ -27,7 +27,7 @@ public Shipment findShipId(int id) {
     /***
      * Saves a Shipment estimate.
      * @param shipment
-     * @return
+     * @return Saved shipment estimate.
      */
```

Comment:
Added return description  JavaDoc. [Ref](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)